kernel/core/datamanagement/tablecolumnimpl.cpp

```diff
@@ -34,6 +36,10 @@ static ConvertibleType detectCommonType(const std::vector<std::string>& vVals)
 {
     ConvertibleType convType = CONVTYPE_NONE;
 
+    last_num_format = 0;
+    for(int i = 0; i < 6; i++)
+        num_format_votes[i] = 0;

```

Comment:
This will simplify to

```suggestion
    int last_num_format = 0; // is actually only used here
    NumberFormatsVoter voter;
```